Headers/DebugServer2/Types.h

```diff
@@ -12,9 +12,9 @@
 #define __DebugServer2_Types_h
 
 #include "DebugServer2/Base.h"
-#include "DebugServer2/CPUTypes.h"
 #include "DebugServer2/Constants.h"
-#include "DebugServer2/ErrorCodes.h"
+#include "DebugServer2/Core/CPUTypes.h"
```

Comment:
I thought you were moving only headers that had an associated source file that moved to `Core/` in the previous commit? `CPUTypes.h` and `ErrorCodes.h` should stay where they are I think?